src/Raven.Studio/wwwroot/App/views/database/indexes/indexErrorDetails.html

```diff
@@ -31,25 +29,23 @@ <h3 class="modal-title">Indexing error details</h3>
                 
                 <div class="details-item">
                     <div class="row">
-                        <div class="col-md-4 text-right">Timestamp</div>
-                        <div class="col-md-8 details-value" data-bind="text: currentError().Timestamp"></div>
+                        <div class="col-md-4 text-right">Time</div>
+                        <div class="col-md-8 details-value">
+                            <span data-bind="text: currentError().LocalTime + '&nbsp;'" class="time-value"></span>
```

Comment:
please remove nbsp - this is not the way we layout elements on screen.

Reply:
removed nbsp